recipe/migrations/root_base6344.yaml

```diff
@@ -0,0 +1,9 @@
+__migrator:
+  build_number: 1
+  commit_message: Rebuild for root_base 6.34.4
+  kind: version
+  migration_number: 1
+migrator_ts: 1740593564.N
```

Comment:
@chrisburr This broke the bot again and this mistake has been committed a migrator before. Please be a bit more careful. :)

Reply:
Ah I'm so sorry! Thanks for fixing it 😄